packages/react/src/Suspensive.tsx

```diff
@@ -14,6 +15,14 @@ export class Suspensive {
   public devMode = new SuspensiveDevMode()
 
   constructor(config: { defaultOptions?: Suspensive['defaultOptions'] } = {}) {
+    if (process.env.NODE_ENV !== 'production') {
+      if (typeof config.defaultOptions?.delay?.ms === 'number') {
+        assert(
+          config.defaultOptions.delay.ms > 0,
+          'Suspensive: config.defaultOptions.delay.ms should be greater than 0'
+        )
+      }
+    }
```

Comment:
But I thought defaultOptions.delay.ms should be > 0. so I made this assertions differently with ms prop of Delay

```suggestion
    if (process.env.NODE_ENV !== 'production') {
      if (typeof config.defaultOptions?.delay?.ms === 'number') {
        assert(
          config.defaultOptions.delay.ms > 0,
          'Suspensive: config.defaultOptions.delay.ms should be greater than 0'
        )
      }
    }
```

Reply:
I made it as variable in assert.ts